tests/queries/tests.py

```diff
@@ -426,14 +473,20 @@ def test_heterogeneous_qs_combination(self):
         with self.assertRaisesMessage(AssertionError, 'Cannot combine queries on two different base models.'):
             Author.objects.all() | Tag.objects.all()
 
-    def test_ticket3141(self):
+    def test_extra_with_count(self):
+        """
+        can't use count() on querysets when extra() has been used with params. (#3141)
```

Comment:
The docstrings are describing how things are were broken rather than  stating the expected behavior such as "Can use count() after extra() with params."